manage.py

```diff
@@ -118,30 +121,35 @@ def pull_prod_data():
     command = "scripts/get_data.sh %s %s" % (prod_db, out_file)
     subprocess.call(shlex.split(command))
 
-    db.session.execute("DELETE FROM ethnicity_in_classification;")
-    db.session.execute("DELETE FROM parent_ethnicity_in_classification;")
-    db.session.execute("DELETE FROM ethnicity;")
-    db.session.execute("DELETE FROM dimension_categorisation;")
-    db.session.execute("DELETE FROM classification;")
-    db.session.execute("DELETE FROM dimension;")
-    db.session.execute("DELETE FROM upload;")
-    db.session.execute("DELETE FROM page;")
-    db.session.execute("DELETE FROM frequency_of_release;")
-    db.session.execute("DELETE FROM lowest_level_of_geography;")
-    db.session.execute("DELETE FROM organisation;")
-    db.session.execute("DELETE FROM type_of_statistic;")
+    for tbl in reversed(db.metadata.sorted_tables):
+        if tbl.name not in inspect(db.engine).get_view_names():
+            print(tbl)
+            db.engine.execute(tbl.delete())
+
+    db.session.execute("DELETE FROM alembic_version")
```

Comment:
Is this not covered bt the previous loop?  Weird.  Although I guess it makes sense that Alembic tries to preserve its own stuff.

Reply:
The `db` metadata looks into sqlalchemy's view of the world, which is all of our models. The `alembic_version` table is added by alembic (duh) rather than sqlalchemy, so the latter doesn't know it exists. Is my understanding.